packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.mdx

```diff
@@ -164,3 +164,109 @@ export const MyComponent = () => {
   return <InteractiveTable columns={columns} data={tableData} getRowId={(r) => r.datasource} />;
 };
 ```
+
+### With pagination
+
+The table can be rendered with pagination controls by passing in the `pageSize` property.
+
+<Story id="experimental-interactivetable--with-pagination" />
```

Comment:
Q: is it worth highlighting that we only support client side pagination?

Reply:
Good idea.